contracts/farms/common/RevoFarmBot.sol

```diff
@@ -88,39 +96,22 @@ contract UbeswapFarmBot is ERC20, AccessControl, Pausable {
 
     address public reserveAddress;
 
-    modifier ensure(uint256 deadline) {
-        require(deadline >= block.timestamp, "FarmBot: EXPIRED");
-        _;
-    }
-
     constructor(
         address _owner,
         address _reserveAddress,
-        address _stakingRewards,
         address _stakingToken,
         address _revoFees,
-        address _swapRouter,
-        address _liquidityRouter,
         address[] memory _rewardsTokens,
```

Comment:
a little weird to me that rewardsTokens would stay, but the stakingRewards contract is removed. seems like they should go together